src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -0,0 +1,49 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.Order;
+import mate.academy.spring.model.dto.response.OrderResponseDto;
+import mate.academy.spring.service.OrderService;
+import mate.academy.spring.service.ShoppingCartService;
+import mate.academy.spring.service.UserService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/orders")
+public class OrderController {
+    private final OrderService orderService;
+    private final UserService userService;
+    private final DtoResponseMapper<OrderResponseDto, Order> orderResponseMapper;
+    private final ShoppingCartService shoppingCartService;
+
+    public OrderController(OrderService orderService,
+                           UserService userService,
+                           DtoResponseMapper<OrderResponseDto, Order> orderResponseMapper,
+                           ShoppingCartService shoppingCartService) {
+        this.orderService = orderService;
+        this.userService = userService;
+        this.orderResponseMapper = orderResponseMapper;
+        this.shoppingCartService = shoppingCartService;
+    }
+
+    @GetMapping
+    public List<OrderResponseDto> allOrberByUser(@RequestParam Long userId) {
```

Comment:
```suggestion
    public List<OrderResponseDto> getByUser(@RequestParam Long userId) {
```

method names should start from verbs